localenv/mock-account-servicing-entity/app/lib/webhooks.server.ts

```diff
@@ -157,6 +157,83 @@ export async function handleIncomingPaymentCompletedExpired(wh: Webhook) {
   return
 }
 
+export async function handleWalletAddressWebMonetization(wh: Webhook) {
+  const walletAddressUrl = wh.data['walletAddressUrl'] as string | undefined
```

Comment:
I don't think we have `walletAddressUrl` on the wh.data on this event. Instead, we have `data.walletAddress.id`. You can check this in the `webhooks.yaml` file (and confirm in the code as well, where we publish `wallet_address.web_monetization` event).

We can then use the `mockAccounts.getByWalletAddressId` function